src/main/java/de/tum/in/www1/artemis/web/rest/dto/ExerciseScoresAggregatedInformation.java

```diff
@@ -0,0 +1,35 @@
+package de.tum.in.www1.artemis.web.rest.dto;
+
+/**
+ * This class ist a container for aggregate information about the achieved scores in exercises
+ */
+public class ExerciseScoresAggregatedInformation {
+
+    public Long exerciseId;
+
+    public Double averageScoreAchieved;
+
+    public Double maxScoreAchieved;
+
+    public ExerciseScoresAggregatedInformation(Long exerciseId, Double averageScoreAchieved, Double maxScoreAchieved) {
+        this.exerciseId = exerciseId;
+        this.averageScoreAchieved = averageScoreAchieved;
+        this.maxScoreAchieved = maxScoreAchieved;
+    }
+
+    public ExerciseScoresAggregatedInformation() {
+        // empty constructor for jackson
+    }
+
+    public Long getExerciseId() {
+        return exerciseId;
+    }
```

Comment:
Does it even make sense to offer getters for `public` variables?

Reply:
You are right! Made them privat b6d37968eccf8608d2e3238b65f233d5cf396252